assets/js/ledyer-checkout-for-woocommerce.js

```diff
@@ -392,9 +390,11 @@ jQuery(function ($) {
                                         // Ledyer will respond with a new event when order is complete
                                         // So don't redirect just yet
                                         lco_wc.isValidating = false;
+                                        lco_wc.redirectUrl = url;
```

Comment:
In `placeLedyerOrder` and if the form was submitted succesfully and validated, everything is done on the WC side. But since we  are in validation mode, we should now wait for Ledyer BE to send us the order complete event before we redirect. So save the redirect url for later, because Ledyer BE could potentially return with an error.